scio-examples/src/main/scala/com/spotify/scio/examples/extra/StatefulExample.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016 Spotify AB.
```

Comment:
🎨 header is a bit off 😄

Reply:
Is there a way to automate header updates?
Also I need to update my IntelliJ settings...